animeal/src/Flows/Main/Coordinator/MainCoordinator.swift

```diff
@@ -13,6 +13,10 @@ enum HomeFlowBackwardAction {
     case shouldShowToast(String)
 }
 
+enum MainFlowSwitchAction {
```

Comment:
@diantyn @ihartsimafeichyk I am not sure about that naming and correct place

Reply:
Looks good for me